blackbox/docs/administration/ssl.txt

```diff
@@ -0,0 +1,117 @@
+.. _administration_ssl:
+
+================================
+Secured Communications (SSL/TLS)
+================================
+
+.. NOTE::
+
+   *Secured Communications* is an Enterprise Edition feature.
+
+Secured communications via SSL allows you to encrypt traffic between CrateDB
+nodes and clients connecting to them. Connections are secured using Transport
+Layer Security (TLS).
+
+
+SSL/TLS Configuration for PostgreSQL wire protocol
+==================================================
+
+First of all you need to set the boolean setting ``ssl.psql.enabled`` to
+``true`` on each of the nodes you want to enable the secured communications.
+
+
+Configure the KeyStore
+----------------------
+
+SSL/TLS needs a keystore. The keystore holds the node certificate(s) which
+should be signed by a certificate authority (CA). A third-party CA or your
+organization's existing CA can be used. When a client connects to a node using
+SSl/TLS the client receives the certificate provided by the node and will
+determine if the node’s certificate is valid, trusted, and matches the hostname
+or IP address it is trying to connect to.
+
+.. NOTE::
+
+    Technically it's possible to use self-signed certificates, but it's
+    strongly recommended to use certificates signed by a CA to establish trust
```

Comment:
i think the point is that it is possible to not check the CA and therefore disable CA checks on the client. what the user needs to do is using a CA that is known by the client too. this can be a private CA in case of a company  PKI  but also official CAs like verisign, it has nothing to do with the fact that it is self-signed